### ISSUE TITLE
Zulip does not start following upgrade to Fedora 35 with NVIDIA GPU

### ISSUE THREAD

**Issue by jdenson12** [zulip/zulip-desktop#1161]
**Describe the bug**

I can't start Zulip following an upgrade to Fedora 35 (was on 33). The following error message is seen in the console:
```
[opt]$ ./Zulip-5.8.1-x86_64.AppImage
Reverting GDK_BACKEND to work around https://github.com/electron/electron/issues/28436
[7110:1109/105526.941773:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
Failed to generate minidump.Illegal instruction (core dumped)
```
It starts fine on my intel GPU laptop, although that is running Wayland.

**To Reproduce**

Fedora 35
NVidia GPU
Running KDE on X11
Try to start Zulip 5.8.1

**Expected behavior**

Zulip should start

**Desktop (please complete the following information):**

- Operating System:
Fedora release 35 (Thirty Five)
5.14.16-301.fc35.x86_64
xorg-x11-drv-nvidia-495.44-3.fc35.x86_64
Running in plasmax11 session

- Zulip Desktop Version:
5.8.1

**Additional context**

I can make it work by adding --no-sandbox to the command:
```
[opt]$ ./Zulip-5.8.1-x86_64.AppImage --no-sandbox
Reverting GDK_BACKEND to work around https://github.com/electron/electron/issues/28436
11:12:28.186 › Checking for update
(node:8721) DeprecationWarning: file property is deprecated and will be removed in v5.
(Use `zulip --trace-deprecation ...` to show where the warning was created)
11:12:28.766 › Update for version 5.8.1 is not available (latest version: 5.8.1, downgrade is disallowed).
```

This was inspired by the issue logged here in another electron based app: 
https://github.com/microsoft/vscode/issues/132455
https://github.com/microsoft/vscode/issues/133593

**Comment by byoungb** [comment]
So the only way I have gotten it to work since upgrading is to run it with the `--no-sandbox` flag, not ideal... but works for now.

**Comment by andersk** [comment]
Based on the linked issues, I expect this is fixed by the Electron upgrade in #1178. Let us know if v5.9.0 doesn't fix this.
